packages/wrangler/package.json

```diff
@@ -36,21 +36,23 @@
     "cli"
   ],
   "dependencies": {
-    "@cloudflare/pages-functions-compiler": "0.3.8",
     "esbuild": "0.14.1",
     "miniflare": "2.0.0-rc.4",
+    "path-to-regexp": "^6.2.0",
```

Comment:
Because it is now also used in the template worker (`lib/functions/template-worker.ts`), which ships as-is. This gets compiled on the user's machine by esbuild.